app/bootstrap.php

```diff
@@ -27,5 +27,17 @@
     ->useAutowiring(false)
     ->build();
 
+if (!file_exists('/usr/local/bin')) {
```

Comment:
@AydinHassan I'm pretty happy with everything except this check. Wanted a simple as possible check to see if we're running on a Unix based system. Something tells me it's not a safe check really.

I don't really want to check for Cygwin because it looks like there is also a whole bunch of other Cywin like apps for Windows which could just get messy. 

Maybe `PHP_OS` to see if it's Windows as with Cygwin `PHP_OS` is  literally just `Cygwin` so I guess other similar apps will do the same